src/libs/core/tests/extractor_test.cpp

```
@@ -0,0 +1,99 @@
+
+    archive_write_close(a);
+    archive_write_free(a);
+    return true;
+}
+} // namespace
+
+class ExtractorTest : public QObject
+{
+    Q_OBJECT
+
+private slots:
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If slots is a macro then please configure it.

(unknownMacro)